event-processing/lambda/models/cleansed-event.ts

```diff
@@ -15,6 +15,21 @@ export interface ICleansedEvent {
     extensions?: unknown | undefined;
```

Comment:
Shouldn't the type of extensions be object?

Reply:
If we change this here, we will need to change every occurrence of extensions in all other code as the type is carried forward.

Reply:
Unknown is correct here. Object type is available at runtime and would limit us only to properties available on the object type. But using unknown tells the compiler it can be anything essentially. It's like using the 'any' type but it's typesafe and is the preferred approach